### PR TITLE
Update Project section

### PR DESCRIPTION
**Pull request by spestana**
These are updates to the Project section of the jupyterbook template, which contains more information outlining what hackweek projects are, how project leaders are expected to go about forming a project, some tips on how to run a project team.

No doubt we'll want to make more changes and add more information as we learn from experience during the 2024 event.

### CODE REVIEW

File: book/projects/project_github.md [hunk 1]
```diff
@@ -25,26 +20,27 @@ We encourage all teams to create a new GitHub repository within the {{hackweek}}
...
 
-### Add the project team to your project repo
+## Add the project team to your project repo
 
 1. Go to the new repository you just created 
-1. Click “Settings” tab in upper right corner
-1. Select “Manage access” on left side of page
-1. Add your team name under the "Manage access" section
-1. Select “write” to enable all team members to make commits
+2. Click “Settings” tab in upper right corner
```
Comment (jomey) [inline book/projects/project_github.md:41]: In markdown land, I think it is preferred to let markdown take care of the numbering and only use `1.` to indicate a numbered list. Frees one up of having to manage the sequence.

File: book/projects/project_before.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+## Project Facilitation Training
+
+Project Facilitation Training will be held a few weeks before the hackweek. The first part of this meeting project leads will give a brief overview of how they are organizing their projects and the mini-tutorials team members might need to work on the project. This time will give projects lead an opportunity to learn from each other and identify any overlapping tutorial needs that might be creatively co-organized.  
+
+During the second half, hackweek organizers will present information on how to effectively facilitate a group project with a diverse team, with varying skills, skill-levels, and learning desires. The goal of this training is to chat about some of the common scenarios that project leads might encounter and offer suggestions, so that leads can feel better equipped and more relaxed to work with what shows up.
+
+
+## Project Development Milestones:
+
+1. **May 21, 2024 - Project Leaders Kick-off Meeting**
```
Comment (jomey) [inline book/projects/project_before.md:35]: Being a template, I wonder how we could generalize all these dates? 🤔

**Comment by spestana** [comment]
Closing this PR, will instead put this content in the [guidebook](https://github.com/uwhackweek/hackweeks-as-a-service/)!

File: book/projects/project_github.md (hunk 1 above)
Comment (spestana) [reply book/projects/project_github.md]: wow! today I learned! thanks :)

**Pull request closed without merging**